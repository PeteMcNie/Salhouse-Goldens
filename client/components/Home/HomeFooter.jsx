import React from 'react'

function HomeFooter () {
  return (
    <div>
      <hr />
      <p>
        Using images from this website
      </p>
      <p>
        All photos on this website are copyright Â© by me. They are creative-commons licensed, you may use them
        under these non-commercial terms.
      </p>
      <p>
        If you want to use them under different terms, please contact me.
      </p>
      <img src='images/nocc.png' />
    </div>
  )
}

export default HomeFooter
