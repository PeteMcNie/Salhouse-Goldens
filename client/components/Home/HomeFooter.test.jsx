import React from 'react'
import { render } from '@testing-library/react'

import HomeFooter from './Home'

test('HomeFooter component to match snapshot', () => {
  const { container } = render(<HomeFooter />)
  expect(container).toMatchSnapshot()
})
