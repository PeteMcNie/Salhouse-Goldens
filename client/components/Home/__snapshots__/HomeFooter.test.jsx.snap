// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeFooter component to match snapshot 1`] = `
<div>
  <nav
    class="navbar navbar-expand-lg navbar-dark bg-dark fixed-top"
  >
    <a
      class="navbar-brand"
      href="/"
    >
      <img
        alt="Goldens retrievers, phoenix and kula"
        class="shlogo d-inline-block align-top"
        src="images/logo.jpeg"
      />
    </a>
    <button
      aria-controls="responsive-navbar-nav"
      aria-label="Toggle navigation"
      class="navbar-toggler collapsed"
      type="button"
    >
      <span
        class="navbar-toggler-icon"
      />
    </button>
    <div
      class="navbar-collapse collapse"
      id="responsive-navbar-nav"
    >
      <div
        class="ml-auto navbar-nav"
      >
        <a
          class="nav-link"
          data-rb-event-key="/"
          href="/"
        >
          Home
        </a>
        <div
          class="dropdown nav-item"
        >
          <a
            aria-expanded="false"
            aria-haspopup="true"
            class="dropdown-toggle nav-link"
            href="#"
            id="collasible-nav-dropdown"
            role="button"
          >
            Our Goldens
          </a>
        </div>
        <div
          class="dropdown nav-item"
        >
          <a
            aria-expanded="false"
            aria-haspopup="true"
            class="dropdown-toggle nav-link"
            href="#"
            id="collasible-nav-dropdown"
            role="button"
          >
            Grooming Your Golden
          </a>
        </div>
        <a
          class="nav-link"
          data-rb-event-key="/mylifewithanimals"
          href="/mylifewithanimals"
        >
          My Life With Animals
        </a>
        <div
          class="dropdown nav-item"
        >
          <a
            aria-expanded="false"
            aria-haspopup="true"
            class="dropdown-toggle nav-link"
            href="#"
            id="collasible-nav-dropdown"
            role="button"
          >
            About Us
          </a>
        </div>
        <a
          class="nav-link"
          data-rb-event-key="/links"
          href="/links"
        >
          Links
        </a>
        <a
          class="nav-link"
          data-rb-event-key="/contact"
          href="/contact"
        >
          Contact
        </a>
      </div>
    </div>
  </nav>
  <div
    class="belowNavbar carousel slide"
  >
    <ol
      class="carousel-indicators"
    >
      <li
        class="active"
      />
      <li />
      <li />
      <li />
    </ol>
    <div
      class="carousel-inner"
    >
      <div
        class="active carousel-item"
      >
        <img
          alt="First slide"
          class="d-block w-100;"
          src="images/lassie.jpg"
        />
        <div
          class="carousel-caption"
        >
          <h3>
            First slide label, do you want this?
          </h3>
          <p>
            Do you want to add further comments below the label?
          </p>
        </div>
      </div>
      <div
        class="carousel-item"
      >
        <img
          alt="Second slide"
          class="d-block w-100"
          src="images/maddie.jpg"
        />
        <div
          class="carousel-caption"
        >
          <h3>
            Second slide label, do you want this?
          </h3>
          <p>
            Do you want to add further comments below the label?
          </p>
        </div>
      </div>
      <div
        class="carousel-item"
      >
        <img
          alt="Third slide"
          class="d-block w-100"
          src="images/munch.jpg"
        />
        <div
          class="carousel-caption"
        >
          <h3>
            Third slide label, do you want this?
          </h3>
          <p>
            Do you want to add further comments below the label?
          </p>
        </div>
      </div>
      <div
        class="carousel-item"
      >
        <img
          alt="Forth slide"
          class="d-block w-100"
          src="images/11.jpg"
        />
      </div>
    </div>
    <a
      class="carousel-control-prev"
      href="#"
      role="button"
    >
      <span
        aria-hidden="true"
        class="carousel-control-prev-icon"
      />
      <span
        class="sr-only"
      >
        Previous
      </span>
    </a>
    <a
      class="carousel-control-next"
      href="#"
      role="button"
    >
      <span
        aria-hidden="true"
        class="carousel-control-next-icon"
      />
      <span
        class="sr-only"
      >
        Next
      </span>
    </a>
  </div>
  <div>
    <h1>
      Welcome to Salhouse Golden Retrievers
    </h1>
  </div>
  <div>
    <p>
      I am a dog lover who fell in love with this beautiful breed when I was a child in England. If you would like to read the story of the animals in my life please go to the My Life with Animals page.
    </p>
    <p>
      The golden retrievers in the above photograph are Tuppence, Shadow, Kula, Sherry and Autumn, with Phoenix laying down in front. Unfortunately our dear sweet Phoenix passed over the Rainbow Bridge in September 2016 at the ripe old age of 14 years, 8 months. However, she is still very much in our hearts and minds and it is very hard to think about the fact that she is no longer with us. My website will still tell the story of Phoenix because she was the most beautiful first golden for me to own and was the reason I wanted to have more of them. I can't thank her enough for the lessons she taught me and the loyalty only a dog can give even when the human is less than perfect.
    </p>
    <p>
      Kula was born in 2003 and is a really loving dog who is my only real retriever at the moment (due either to the other four's obstinance or my bad training). She loves nothing more than to swim in the river or sea fetching as many sticks as I can throw for her. In 2007, Kula had a beautiful litter of puppies and we kept one female who we named Shadow. She is a real character and is such fun to be with. She is so enthusiastic about her people and yet she has the loveliest nature. Shadow had a litter of 7 puppies in April 2010 and we have kept one of those, Sherry. We added an item to the News page each day from the day the puppies were born to the time they left us. There are loads of photos and videos of these puppies and some stories of them growing up. Go to the News page to see these stories and more. Sherry gave birth to two female puppies on 16 June 2015 - Autumn and Tuppence. Only having two in the litter made it impossible for us to part with one, so, although we had only wanted to keep one, we found ourselves with the two naughties! And then on from that episode, Autumn had a litter of seven puppies in March 2019, of which we kept one who we called Siren.
    </p>
    <p>
      If you want to find out more about each of these beautiful dogs, please go to their page to learn more.
    </p>
    <p>
      Shadow, Sherry and Autumn are breed champions. Sherry did compete in championship obedience competitions up to Test A level and has gained the CDX title. However, when Autumn and Tuppence came along it was too much to continue training Sherry and the puppies and so she is retired from the obedience ring. Tuppence is a very smart dog and at the present time is in training and hopefully one day I will get her into the obedience ring. Go to their individual pages to learn more about them.
    </p>
    <p>
      Goldens are wonderful pets - they are loyal, friendly and good with children . I hope you like mine and get your own one day!
    </p>
    <p>
      This website has lots of lovely photos of goldens and some other animals that I have owned, I hope you enjoy looking round it. Click on each photo if you want to see it enlarged.
    </p>
  </div>
  <div>
    <hr />
    <p>
      Using images from this website
    </p>
    <p>
      All photos on this website are copyright Â© by me. They are creative-commons licensed, you may use them under these non-commercial terms.
    </p>
    <p>
      If you want to use them under different terms, please contact me.
    </p>
    <img
      src="images/nocc.png"
    />
  </div>
</div>
`;
